src/core/a-entity.js

```diff
@@ -246,33 +248,32 @@ var proto = Object.create(ANode.prototype, {
    * Initialize component.
    */
   initComponent: {
-    value: function (name, isDependency) {
+    value: function (name, isDependency, attr) {
```

Comment:
It would help to rename `attr` to `attrValue`. I got confused and thought it might be an `attrName`.